tests/test_indexes.py

```diff
@@ -231,3 +231,34 @@ async def test_index_05(self):
                 };
                 """
             )
+
```

Comment:
We also need some tests for creating new abstract indexes via extending, right?
(Though that's definitely lower priority than FTS, so if it turns out to be pretty busted and tricky to get working we might be able to punt on that for now)